fe1-web/src/features/wallet/network/index.ts

```diff
@@ -0,0 +1,12 @@
+import { KeyPairRegistry } from 'core/keypair/KeyPairRegistry';
+import { SignatureType } from 'core/network/jsonrpc/messages';
+import { getCurrentPopTokenFromStore } from 'features/wallet/objects';
+
+/**
+ * Configures the network callbacks in a KeyPairRegistry.
+ *
+ * @param registry - The KeyPairRegistry where we want to add the mapping
+ */
+export function configureNetwork(registry: KeyPairRegistry) {
+  registry.add(SignatureType.POP_TOKEN, getCurrentPopTokenFromStore);
```

Comment:
this isn't really related to `network` . I would consider just adding this one line in `wallet/index.ts` for now.